std/internal/math/biguintcore.d

```diff
@@ -52,6 +52,130 @@ import std.range.primitives;
 import std.traits;
 
 private:
+
+// compile-time & run-time dipatchers to allows BigInt CTFE for 32 bit OS.
+
+pragma(inline, true)
+uint multibyteAddSub(char op)(uint[] dest, const(uint)[] src1, const (uint)[] src2, uint carry) pure
+{
+    if (__ctfe)
```

Comment:
Does are the times where CTFE + `static if` would be really cool:

```
static if(__ctfe || isNotX86)
```

Reply:
Yes. I remember that Stefan said that it could be possible, so it may be true in the future.